stable/spinnaker/values.yaml

```diff
@@ -1,8 +1,8 @@
 halyard:
-  spinnakerVersion: 1.9.1
+  spinnakerVersion: 1.10.11
```

Comment:
```suggestion
  spinnakerVersion: 1.13.1
```

Reply:
I based the version to use off https://www.spinnaker.io/community/releases/versions/#latest-stable which shows `1.10.11` as the latest stable version of spinnaker, whereas [1.13.1](https://console.cloud.google.com/gcr/images/spinnaker-marketplace/GLOBAL/halyard?gcrImageListsize=30&gcrImageListquery=%255B%255D) is the latest stable version of halyard.   I actually had tried `1.13.1` for spinnaker and it failed with strange errors about not having permission to access the bill of materials, which turned out to be because there was no bom for `1.13.1` :cry:

Reply:
so we're using halyard 1.13.1 to install spinnaker 1.10.11.  I didn't see anything to suggest that the two versions were meant to match (and there isn't a halyard 1.10.11, which further suggests this) so I think this is ok.

Reply:
@paulczar thanks for the clarification, for this is a bit confusing :(

However, I saw in the release page it have the 1.11.6 we can use that? Or the 1.10.X is the better option?  https://www.spinnaker.io/community/releases/versions/#version-1116

Reply:
omg yes we can!  I just looked at the releases page in top-to-bottom order and took the top :laughing: